DevelopGitHub/GitHubExtension/GitHubExtension.Templates/Controllers/TemplatesCommandController.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Templates.Commands;
+using GitHubExtension.Templates.CommunicationModels;
+using GitHubExtension.Templates.Constants;
+using GitHubExtension.Templates.ExtensionMethods;
+
+namespace GitHubExtension.Templates.Controllers
+{
+    [RoutePrefix(RouteTemplatesConstants.GetGitHubTemplatesRoute)]
+    public class TemplatesCommandController : ApiController
+    {
+        private readonly ITemplatesCommand _templatesCommand;
+        private const string PathToPullRequestTemplate = ".github/PULL_REQUEST_TEMPLATE.md";
+        private const string PathToIssueTemplate = ".github/ISSUE_TEMPLATE.md";
+        private const string NotOwner = "notOwner";
+        private const string NotOwnerError = "You are not an owner of repository";
+
+        public TemplatesCommandController(ITemplatesCommand templatesCommand)
+        {
+            _templatesCommand = templatesCommand;
+        }
+
+        [Authorize]
+        [HttpPut]
+        [Route(RouteTemplatesConstants.CreatePullRequestTemplate)]
+        public async Task<IHttpActionResult> CreatePullRequestTemplate([FromBody]CreateTemplateModel model)
```

Comment:
Your method name is CreatePullRequestTemplate but you are using HttpPut so you are planning create or update using this method?